public/locales/cy/translations.json

```
@@ -96,37 +96,37 @@
   "follow on": "Rydych chi'n dilyn",
   "follow on efp": "Rydych chi eisoes yn dilyn EFP",
   "to add": "Cyfanswm i'w ychwanegu at y drol",
-  "next": "Nesaf",
-  "finish": "Gorffen",
+  "transaction.next": "Nesaf",
+  "transaction.finish": "Gorffen",
```

Biome (1.9.4)

[error] The key transaction.finish was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)